src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml.cs

```diff
@@ -155,6 +155,7 @@ void OnWorkspaceViewUnloaded(object sender, RoutedEventArgs e)
             if (ViewModel != null)
             {
                 removeViewModelsubscriptions(ViewModel);
+                ViewModel.RequestNodeAutoCompleteSearch -= ShowHideNodeAutoCompleteControl;
```

Comment:
Not sure how I missed this, but this line can maybe placed as part of `removeViewModelsubscriptions()`